catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/CsdidaAttribute.java

```diff
@@ -25,39 +25,130 @@
 import org.codice.imaging.nitf.core.tre.Tre;
 
 import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.BasicTypes;
 
 /**
  * TRE for "Dataset Identification"
  */
-class CsdidaAttribute extends NitfAttributeImpl<Tre> {
+public class CsdidaAttribute extends NitfAttributeImpl<Tre> {
 
     private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
 
-    public static final String PLATFORM_CODE = "PLATFORM_CODE";
+    public static final String PLATFORM_CODE_NAME = "PLATFORM_CODE";
 
-    public static final String VEHICLE_ID = "VEHICLE_ID";
+    public static final String VEHICLE_ID_NAME = "VEHICLE_ID";
+
+    public static final String DAY_NAME = "DAY";
+
+    public static final String MONTH_NAME = "MONTH";
+
+    public static final String YEAR_NAME = "YEAR";
+
+    public static final String PASS_NAME = "PASS";
+
+    public static final String OPERATION_NAME = "OPERATION";
+
+    public static final String SENSOR_ID_NAME = "SENSOR_ID";
+
+    public static final String PRODUCT_ID_NAME = "PRODUCT_ID";
+
+    public static final String TIME_NAME = "TIME";
+
+    public static final String PROCESS_TIME_NAME = "PROCESS_TIME";
+
+    public static final String SOFTWARE_VERSION_NUMBER_NAME = "SOFTWARE_VERSION_NUMBER";
+
+    private static final String ATTRIBUTE_NAME_PREFIX = "csdida.";
 
     public static final CsdidaAttribute PLATFORM_ID = new CsdidaAttribute(Isr.PLATFORM_ID,
```

Comment:
The constructor used by the taxonomy normalized attributes also supports the non-taxonomy mapping.